schema/objects/Financing.schema.json

```diff
@@ -13,11 +13,12 @@
     "object_type": {
       "const": "FINANCING"
     },
+    "id": {},
+    "comments": {},
```

Comment:
minor: the common pattern is to have the repeated property names (anything with empty `{}`) at the _bottom_ of the property list